starter-code/index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(x) {
+    this.items.push(x);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.length || pos < 0) {
+      throw new Error('OutOfBounds');
+    } else {
```

Comment:
The else statement is not needed here. You can just write: return this.items[pos]